packages/carbon-web-components/src/components/list/ordered-list.stories.ts

```diff
@@ -0,0 +1,138 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2019, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html } from 'lit';
+import storyDocs from './list.mdx';
+import './index';
+
+const defaultArgs = {
+  isExpressive: false,
+  native: false,
```

Comment:
```suggestion
  native: true,
```

I think this should be `true` by default.